configapi/api_default.go

```diff
@@ -47,7 +53,15 @@ func GetDeviceGroups(c *gin.Context) {
 	c.JSON(http.StatusOK, deviceGroups)
 }
 
-// GetDeviceGroupsByName -
+// GetDeviceGroupsByName godoc
```

Comment:
```suggestion
// GetDeviceGroupByName godoc
```

Reply:
This is not a big deal and does not appear but we can keep it same with the method name.